packages/roosterjs-content-model/lib/publicTypes/format/ContentModelSegmentFormat.ts

```diff
@@ -13,6 +14,7 @@ import { UnderlineFormat } from './formatParts/UnderlineFormat';
  * The format object for a segment in Content Model
  */
 export type ContentModelSegmentFormat = TextColorFormat &
+    DirectionFormat &
```

Comment:
Why need to add DirectionFormat to SegmentFormat?

Reply:
We need it so it can use to set alignment in the formatHolder

Reply:
In that case, this format should not be part of the formatHolder since formatHolder only handles inline format. (I realized that using a selectionMarker as format holder may not be a good idea, let me change this in separate change).

Actually ListItem.format is already ContentModelBlockFormat which contains DirectionFormat